scitbx/array_family/boost_python/tst_flex.py

```diff
@@ -1113,8 +1112,15 @@ def exercise_operators():
   assert a.all_le(9)
   assert not a.all_ge(b)
   assert a.all_ge(2)
-  assert flex.order(a, b) == cmp(tuple(a), tuple(b))
-  assert flex.order(b, a) == cmp(tuple(b), tuple(a))
+  if tuple(a) < tuple(b):
+      assert flex.order(a, b) == -1
+      assert flex.order(b, a) == 1
+  elif tuple(a) > tuple(b):
+      assert flex.order(a, b) == 1
+      assert flex.order(b, a) == -1
+  else:
+      assert flex.order(a, b) == 0
+      assert flex.order(b, a) == 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.